README.md

```diff
@@ -1,2 +1,15 @@
 # ion-blender-exporter
 Blender add-on for uploading and tiling models with Cesium ion. https://cesium.com
+
+## Installation
+1. Download the project as a zip [here](https://github.com/AnalyticalGraphicsInc/ion-blender-exporter/archive/beta.zip)
```

Comment:
This is OK for now, but we'll need to figure out the release process, I assume we will eventually link to the latest official releases taged with GitHub releases.

Reply:
Yep

Reply:
TODO: Switch this to using releases instead & Write release documentation